core-web/libs/sdk/react/src/lib/hooks/useCheckHaveContent.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useRef, useState } from 'react';
+
+export const useCheckHaveContent = () => {
+    const contentletDivRef = useRef<HTMLDivElement | null>(null);
+    const [haveContent, setHaveContent] = useState<boolean>(false);
+
+    useEffect(() => {
+        if (!contentletDivRef.current) {
+            return;
+        }
+
+        const childElement = contentletDivRef.current.firstElementChild;
+
+        if (!childElement) {
+            return;
+        }
+
+        const height = childElement.getBoundingClientRect().height;
+
+        if (height > 0) {
```

Comment:
```suggestion
        if (height > 10) {
```

Maybe?

Also... does this looks like the traditional?

Reply:
height > 0 is the same as traditional, that covers the case if the content is empty (no html) or have html without code inside (like <p><span></span></p>

And yes, now looks like traditional

traditional:
![Screenshot 2024-09-25 at 10 59 40 AM](https://github.com/user-attachments/assets/b30f8ec8-a499-4255-acb1-8db6d95ee576)

Headless:
![Screenshot 2024-09-25 at 10 59 12 AM](https://github.com/user-attachments/assets/c0981979-75e1-48f3-b6e4-0a829dc52585)